tidb-cloud/configure-maintenance-window.md

```diff
@@ -0,0 +1,100 @@
+
+5. To defer the start time of a scheduled maintenance activity, click **Defer** in the **Action** column and change it to the next feasible window before the deadline.
+
+    If you need to defer the maintenance activity beyond the deadline, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md#tidb-cloud-support) for assistance.
+
+## FAQs
+
+- What are maintenance activities?
+
+    Maintenance activities typically include operating system updates, security patches, and infrastructure upgrades.
+
+- How often does maintenance occur?
+
+    Maintenance is infrequent, usually occurring once every few months. The maintenance activities will be displayed on the **Maintenance** page several days before the scheduled time and you will receive an email notification 72 hours in advance, allowing you to make necessary preparations.
+
+- Can I disable a maintenance window?
+
+    The maintenance window is enabled by default and cannot be disabled. You can modify the start time of the maintenance window or defer a maintenance activity to 2 to 4 weeks until the deadline. For more information, see [View and configure maintenance windows](#view-and-configure-maintenance-windows).
+
+- How long does a maintenance window last?
+
+    For each project, maintenance is executed on eligible TiDB clusters one by one. The duration of maintenance varies depending on the number of clusters, cluster data size, and the maintenance activities to be performed.
+
+- Will maintenance activities be performed on clusters in any status?
+
+    No. TiDB Cloud will check the cluster status before performing a maintenance activity on a cluster.
+
+    - If the cluster is in the **Creating** or **Paused** status, maintenance activities are not required.
+    - If the cluster is in the **Backup** status, the maintenance will be delayed and triggered until the current backup is successfully completed. Note that for clusters with large data volumes, the backup process might take a long time, such as 12 hours. To minimize the impact on the clusters, it is recommended to carefully set the start time for backups and the maintenance window.
```

Comment:
“If the cluster is in the Backup status”：Cluster has no Backup status . it is possible to say that the cluster is currently running a backup task.